codes/go/chapter_array_and_linkedlist/list_test.go

```diff
@@ -0,0 +1,67 @@
+// File: list_test.go
+// Created Time: 2022-12-18
+// Author: msk397 (machangxinq@gmail.com)
+
+package chapter_array_and_linkedlist
+
+import (
+	"fmt"
+	"sort"
+	"testing"
+)
+
+/* Driver Code */
+func TestList(t *testing.T) {
+	/* 初始化列表 */
+	list := []int{1, 3, 2, 5, 4}
+	fmt.Println("列表 list =", list)
+
+	/* 访问元素 */
+	num := list[1] // 访问索引 1 处的元素
+	fmt.Println("访问索引 1 处的元素，得到 num =", num)
+
+	/* 更新元素 */
+	list[1] = 0 // 将索引 1 处的元素更新为 0
+	fmt.Println("将索引 1 处的元素更新为 0 ，得到 list =", list)
+
+	/* 清空列表 */
+	list = []int{}
```

Comment:
I think `list = []int{}` is to initialize but not clear a slice. It'd be better to use `list = nil` instead,

Reply:
You are right, I have finished modifying!